src/components/TermsFooter.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React, { useCallback } from "react";
+import { Linking, StyleSheet } from "react-native";
+import { urls } from "../config/urls";
+import LText from "./LText";
+
+function TermsFooter({
+  t,
+  provider,
+}: {
+  t: (key: string, options?: { [key: string]: string | number }) => string,
+  provider: string,
+}) {
+  const url = urls.swap.providers[provider]?.tos;
+  const onLinkClick = useCallback(() => Linking.openURL(url), [url]);
+
+  if (!url) {
+    return null;
+  }
+
+  return (
+    <>
+      <LText style={[styles.text]}>
+        {t("DeviceAction.confirmSwap.acceptTerms")}
+      </LText>
+      <LText style={[styles.text, styles.link]} onPress={onLinkClick}>
+        {t("DeviceAction.confirmSwap.termsLink", { provider })}
```

Comment:
In order to capitalize the provider (which is something product was super worried about) you can do something like  we did for the `transfer.swap.form.summary.disclaimer.desc` wording. Basically something like
```
<Trans
  i18nKey={"DeviceAction.confirmSwap.termsLink"}
  values={{ provider }}
>
  <LText style={styles.capitalize} color="smoke" />
</Trans>
```

And the wording changing to 
`<0>{provider}}</0>'s terms of use`

Reply:
Perfect thanks for the example, I knew this syntax with i18next but I didn't think of it

Reply:
Done